config/locales/views/admin/en.shared.yml

```
@@ -2,4 +2,4 @@ en:
   views:
     shared:
       statements_table:
-        no_statements: No statements match your criteria
+        no_statements: 'No !statement_downcase! match your criteria'
```

I lean towards " over ' - also this one should be plural I think
```suggestion
        no_statements: "No !statements_downcase! match your criteria"
```